src/Auth/project.json

```diff
@@ -16,7 +16,8 @@
     "Npgsql.EntityFrameworkCore.PostgreSQL": "1.0.0-rc2-release1",
     "Npgsql.EntityFrameworkCore.PostgreSQL.Design": "1.0.0-rc2-release1",
     "Npgsql": "3.1.1",
-    "OpenIddict": "1.0.0-*",
+    "OpenIddict.Core": "1.0.0-alpha2-0288",
```

Comment:
There are still a few things to implement before releasing a first "stable" version.
I hope we'll be able to ship it in a month.